common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -457,7 +458,8 @@
             var completionUrl = this.ajaxUrl + '/get_completion';
             var usageKey = element[0].attributes['data-id'].value;
             var completionIndicators = element.find('.check-circle');
-            var element_title = element[0].getAttribute("data-page-title");
+            var $progressPercentage = $('#progress-percentage');
```

Comment:
Remove `$` from the variable name to match the naming convention of the existing codebase.